packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/InputAwareWebView.java

```diff
@@ -0,0 +1,102 @@
+package io.flutter.plugins.webviewflutter;
+
+import static android.content.Context.INPUT_METHOD_SERVICE;
+
+import android.content.Context;
+import android.view.View;
+import android.view.inputmethod.InputMethodManager;
+import android.webkit.WebView;
+
+/**
+ * A WebView subclass that mirrors the same implementation hacks that the system WebView does in
```

Comment:
I will be a little more specific here, mention that these hacks happen only prior to Android N, and this view is here to support these Android versions. I'd mention that the point of the workaround is to create an input connection on a dedicated thread.